suite-native/device-manager/src/components/DeviceItem/DeviceItemContent.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { HStack, Box } from '@suite-native/atoms';
+import { Translation, useTranslate } from '@suite-native/intl';
+import {
+    selectDeviceByState,
+    DeviceRootState,
+    PORTFOLIO_TRACKER_DEVICE_ID,
+} from '@suite-common/wallet-core';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { DeviceItemIcon } from './DeviceItemIcon';
+import { SimpleDeviceItemContent } from './SimpleDeviceItemContent';
+import { WalletDetailDeviceItemContent } from './WalletDetailDeviceItemContent';
+import { DeviceItemContentProps } from '../../types';
+
+const contentWrapperStyle = prepareNativeStyle(_ => ({
+    flexShrink: 1,
+    height: 56,
+}));
+const itemStyle = prepareNativeStyle(_ => ({
+    flexShrink: 1,
+}));
+
+export const DeviceItemContent = ({
+    deviceState,
+    isPortfolioLabelDisplayed = true,
+    headerTextVariant = 'body',
+    iconSize = 'large',
+    variant = 'simple',
+}: DeviceItemContentProps) => {
+    const { translate } = useTranslate();
+    const { applyStyle } = useNativeStyles();
+
+    const device = useSelector((state: DeviceRootState) =>
+        deviceState !== undefined ? selectDeviceByState(state, deviceState) : undefined,
+    );
+
+    const isPortfolioTrackerDevice = device?.id === PORTFOLIO_TRACKER_DEVICE_ID;
+
+    const deviceHeader =
+        (isPortfolioTrackerDevice ? device?.name : device?.label) ??
+        translate('deviceManager.defaultHeader');
+
+    const walletNameLabel = device?.useEmptyPassphrase ? (
+        <Translation id="deviceManager.wallet.standard" />
+    ) : (
+        <Translation
+            id="deviceManager.wallet.defaultPassphrase"
+            values={{ index: device?.walletNumber }}
+        />
+    );
+
+    return (
+        <HStack alignItems="center" spacing="medium" style={applyStyle(contentWrapperStyle)}>
+            <DeviceItemIcon deviceId={device?.id} iconSize={iconSize} />
```

Comment:
```suggestion
            <DeviceItemIcon deviceId={deviceState} iconSize={iconSize} />
```

Reply:
icon is not based on certain wallet but instead physical model, so there is no need to pass state instead of id which I also need here in this component.

Reply:
Okay, then please just get rid of the `?` in `device?.id`